Assets/CharTest/Scripts/SeleneFire.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SeleneFire : MonoBehaviour
+{
+
+    [SerializeField]
+    private int primaryProjectiles;     //Number of projectiles in first wave of primary spell
+    [SerializeField]
+    private int primaryWaves;           //Number of waves in primary spell
+    [SerializeField]
+    private float primarySpeed;         //Speed of projectiles in primary spell
+    [SerializeField]
+    private int heavyProjectiles;       //Number of split heavy projectiles
+
+    public GameObject primaryPrefab;  
+    public GameObject heavyPrefab;
```

Comment:
These can be
[SerializeField]
private ...

There are other parts of your code that could be changed like this. Have a look around.